composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.7@dev || ^7.0@dev",
+        "ezsystems/ezpublish-kernel": "~6.7.7 || ^6.12.1 || ^7.0",
```

Comment:
you'll need to keep dev for tests to continue to work

Reply:
fixed in 358a682

Reply:
Fixed again in b8a23482497c7925fde9f844b5c39a54c74ee357

please pull in changes before rebasing ;)